### DISCUSSION TITLE
Example of mapURLs

### DISCUSSION THREAD

**Discussion by arthurrmp** [google/model-viewer#2041] [Q&A] [answered]
### Description

Please provide an example of the method mapURLs. I'm trying to use it but I'm getting the error "....mapURLs is not a function".

#### Live Demo

https://glitch.com/edit/#!/florentine-piquant-lasagna

### Version

- model-viewer: v1.5.0

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [X] Windows

**Comment by elalish** [comment]
It's because it's a static method, which needs to be called differently, as shown in this [example](https://modelviewer.dev/examples/loading/#dracoSupport).

**Comment by arthurrmp** [comment, marked as answer]
I've been able to use it like that. 

```
const ModelViewerElement = customElements.get('model-viewer');
ModelViewerElement.mapURLs((url) => {
      // Do the modifications you need...
      return newUrl;
}); 
```
